mobile/status.go

```diff
@@ -351,6 +351,11 @@ func SaveAccountAndLogin(accountData, password, settingsJSON, configJSON, subacc
 	if err != nil {
 		return makeJSONResponse(err)
 	}
+
+	if *settings.Mnemonic != "" {
+		settings.OmitTransfersHistoryScan = true
```

Comment:
@rasom could you point me to the piece of code where we're doing the history scan if new accounts are added on a later date, after the user backed up his mnemonic?

Reply:
This change is ok, and that's the initial state when a user is creating a regular (non-keycard) Status account.
But then the user can back up the mnemonic, use the same seed phrase in other tools, then get back to Status and add the same account he used elsewhere. Do we handle that case?

Reply:
We don't want to scan history right after account creation. It doesn't matter what happens with mnemonic afterwards in terms of this change.

Reply:
Aha... so it's only for the default wallet account?

Reply:
But anyway, I don't see code that avoids history scans based on `OmitTransfersHistoryScan` (not in this PR, but in `status-go` repo), do we have it or not yet?

Reply:
Ok, I see this https://github.com/status-im/status-go/pull/4163/files#diff-856ac71f99a073eb293c1213dc5ad2e9e620b0174fdc369cd39951d50a5117b4R106-R112 
thanks for sharing that